test/conformance/README.md

```diff
@@ -0,0 +1,8 @@
+# Conformance tests
+
```

Comment:
maybe it's an obvious statement, but I'd add info that "when you fix tests you have to update match files" and perhaps mention why we have empty match files...?